hads.js

```diff
@@ -118,11 +120,21 @@ app.get('*', (req, res, next) => {
       title = search ? renderer.searchResults : path.basename(filePath);
     }
 
+    if (!lastModified) {
+      fs.stat(filePath, (err, stats) => {
```

Comment:
`fs.stat` is asynchronous, so there is a race condition here and on the other occurence below. You should use `fs-extra` to make it resolve to a promise and chain it with the `contentPromise`

Reply:
OK I will try to resolve that asap.

Sorry for my JS coding that is certainly not as good as I would like to ;-)

Reply:
I have fixed my issues.

Do I have to recreate a new pull request? Or how can I update this PR?

Reply:
If you push your changes the PR will be automatically updated